File: 2025/python/utils/output_utils.py



def print_output(val: str, use_test_data: bool = False):
	outputString = ""
	if use_test_data:
		outputString = "Total [using test data]"
	else:
		outputString = "Total [using puzzle data]"
	
	print(f"{outputString}: {val}")

File: 2025/go/utils/output_utils.go
package utils

import (
	"fmt"
)

func PrintOutput(val string, useTestData bool) {
	var outputString = "Total [using puzzle data]: "

	if useTestData {
		outputString = "Total [using test data]: "
	}

	fmt.Println(outputString, val, "\n\n")
}

func PrintMatrix(matrix [][]string) {
	for _, row := range matrix {
		for _, val := range row {
			fmt.Print(val)
		}
		fmt.Println()
	}
}
